### DISCUSSION TITLE
There is one element missing in the connection methods

### DISCUSSION THREAD

**Discussion by AMD-NICK** [iterate-ch/cyberduck#16292] [Q&A] [answered]
I want to connect to my backblaze storage. It is located at s3.**us-west-004**.backblazeb2.com, but this exact adapter is missing from the list

<img width="599" alt="image" src="https://github.com/user-attachments/assets/9a82d116-cf86-4195-92f8-b6905d368819">

Cyberduck 8.9.0 (41543)
MacOS Sonoma 14.1.2 (23B92) (M1 chip)

**Comment by dkocher** [comment, marked as answer]
You can connect using the regular _Backblaze B2 Cloud Storage_ option available by default with no additional connection profile required that will use the B2 API. The [additional S3 connection profiles](https://docs.cyberduck.io/protocols/b2/#using-s3-compatible-api) are only available if for some reason a connection using the S3 compatibility layer is needed.
